MekHQ/unittests/mekhq/campaign/parts/equipment/LargeCraftAmmoBinTest.java

```diff
@@ -276,6 +276,7 @@ public void getBayFallsBackToMatchingWeaponBayIfNoBayEquipment() {
         weaponBays.add(wrongWeaponBay);
         weaponBays.add(weaponBay);
         when(entity.getWeaponBayList()).thenReturn(weaponBays);
+        when(entity.whichBay(equipmentNum)).thenReturn((WeaponMounted) weaponBay);
```

Comment:
I don't *love* this solution because `entity.whichBay` should already work because it calls `entity.getWeaponBayList` which is mocked in line 278. However, on the debugger, that method was returning null within the `whichBay` function. I think its ok?